2024-01-20 10:58:30.261 | ERROR    | pickup:urgePickUp:24 - 捕获到异常：HTTPSConnectionPool(host='api-seller.ozon.ru', port=443): Max retries exceeded with url: /v3/posting/fbs/list (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1131)')))) 异常类型：<class 'requests.exceptions.ProxyError'>
